homeassistant/components/fritzbox/sensor.py

```diff
@@ -17,6 +15,8 @@
 )
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import (
+    ELECTRIC_CURRENT_AMPERE,
+    ELECTRIC_POTENTIAL_VOLT,
```

Comment:
every change in the import section -besides these to additions (_`ELECTRIC_CURRENT_AMPERE` and `ELECTRIC_POTENTIAL_VOLT`_)- should not be there

Reply:
```
root ➜ /workspaces/homeassistant-core (dreimer1986-patch-1) $ isort homeassistant/components/fritzbox/sensor.py 
Fixing /workspaces/homeassistant-core/homeassistant/components/fritzbox/sensor.py
```